t/python/t0013-job-list.py

```diff
@@ -74,170 +74,266 @@ def waitForConsistency(self, jobs_list_length):
             if waiting > 60:
                 raise TimeoutError()
 
-    attrs = ["userid", "state", "name", "ntasks", "t_submit", "t_run", "t_inactive"]
+    # flux job list should return an empty list if there are no jobs
+    def test_00_list_expect_empty_list(self):
+        rpc_handle = flux.job.job_list(self.fh)
 
-    # should return an empty list if there are no inactive jobs
-    def test_00_list_inactive_expect_empty_list(self):
-        rpc_handle = flux.job.job_list_inactive(
-            self.fh, time.time() - 3600, 10, self.attrs
-        )
+        jobs = self.getJobs(rpc_handle)
+
+        self.assertEqual(len(jobs), 0)
+
+    # flux job list-inactive should return an empty list if there are
+    # no inactive jobs
+    def test_01_list_inactive_expect_empty_list(self):
+        rpc_handle = flux.job.job_list_inactive(self.fh, time.time() - 3600, 10)
 
         jobs = self.getJobs(rpc_handle)
 
         self.assertEqual(len(jobs), 0)
 
-    # make sure one job is read from RPC
-    def test_01_list_inactive_success(self):
+    core_attrs = (
+        {
+            "id",
+            "userid",
+            "urgency",
+            "priority",
+            "state",
+            "t_submit",
+            "t_run",
+            "t_inactive",
+            "name",
+            "ranks",
+        },
+    )
```

Comment:
What you have here is a 1-tuple with a single element, a set. I don't think the tests will pass because `self.assertLessEqual(core_attrs, set(job.keys()))` should give you something like `TypeError: '<' not supported between instances of 'tuple' and 'set'` unless I'm missing something

Reply:
If you removed the parentheses and the comma I think it should work. Technically you could just remove the comma but I think better to remove the parentheses too

Reply:
oops!  I didn't push my last changes :P  somehow `black` didn't like my formatting and did that.